### ISSUE TITLE
Zoom Out?

### ISSUE THREAD

**Issue by perlmunger** [evgenyneu/ios-imagescroll-swift#7]
First off thanks for this example code. It's the most concise and best solution I've seen for getting zooming to work consistently with auto layout. I really appreciate.

So, I'm probably just missing something simple, but when I pinch in and release, the image view just snaps back to its starting position. I thought this was controlled by changing the minimum zoom scale, but when I mess with that in updateZoom(), it doesn't work the way I expect. 

In other words, how can I make the example zoom out as well as in?

Thanks again.

**Comment by evgenyneu** [comment]
Hi @perlmunger, thank you for kind words and a good question.

Just to make sure I understood you correctly, do you want to be able to keep the image zoomed out to make it smaller than scroll view (see the screenshot below)?

<img width="169" alt="swift_imagescroll" src="https://cloud.githubusercontent.com/assets/880411/20280827/30f0de56-ab02-11e6-94e2-cb260a0b869f.png">

If so, one of way of doing it is to look at the `updateZoom` function and add a multiplier to the following code, a `0.5` for example:

``` Swift
scrollView.minimumZoomScale = 0.5 * minZoom
```

By default, the code sets the minimum zoom scale, such that one can not zoom out to make the image smaller than the view. However, as shown in the example above, one can set a smaller minimum zoom scale. Is it what you were asking?

**Comment by perlmunger** [comment]
@evgenyneu That's exactly what I was asking! Perfect. That makes sense now. Thank you again. I appreciate it.
